test/distributed/org/apache/cassandra/distributed/test/GossipTest.java

```diff
@@ -277,6 +279,13 @@ public void gossipShutdownUpdatesTokenMetadata() throws Exception
         }
     }
 
+    private static String getLocalToken(IInvokableInstance node)
+    {
+        Collection<String> tokens = ClusterUtils.getLocalTokens(node);
+        Assertions.assertThat(tokens).hasSize(1);
+        return tokens.stream().findFirst().get();
+    }
+
```

Comment:
used to be `ClusterUtils.getLocalToken` but that is now `Collection<String>` so can't use it without updating the code to handle it... so simplify to match the old test.

Reply:
Yeah, was wondering if this was more general purpose useful (i.e. should live in some kind of Utils class) rather than restricting it to Gossip only.

Might make sense to move this over adjacent to `ClusterUtils.getLocalTokens` as a general purpose helper function even though we only use it in this one place atm. Otherwise we're more likely to duplicate this logic over there in the future when someone needs it if they don't dig enough to find this.

Reply:
so, if we code against 1 token then the test can't really support vnode, so find it best to not generally support a single token API for that reason

Reply:
Hm. Good point. Promoting this would effectively encourage that behavior by virtue of making it lower friction to engage in.

WFM. Carry on.